testsuite/performance/README.md

```diff
@@ -70,47 +70,74 @@ Provisioning/teardown is performed via `docker-compose` tool. More details in [R
+
+
 #### Examples:
 - `mvn verify -Pgenerate-data` - generate default dataset
 - `mvn verify -Pgenerate-data -DusersPerRealm=5` - generate default dataset, override the `usersPerRealm` property
 - `mvn verify -Pgenerate-data -Ddataset=100u` - generate `100u` dataset
 - `mvn verify -Pgenerate-data -Ddataset=100r/default` - generate dataset based on `datasets/100r/default.properties`
 
-The data can also be exported from the database, and stored locally as `datasets/${dataset}.sql.gz`
-`DATASET=100u ./prepare-dump.sh`
+#### Export / Import Database Dump
 
 To speed up dataset initialization part, it is possible to pass `-Dexport-dump` option to have the generated dataset
 exported right after it has been generated. Then, if there is a data dump file available then `-Pimport-dump` 
 can be used to import the data directly into the database, bypassing Keycloak server completely.
 
-Usage: `mvn verify -Pimport-dump [-Ddataset=DATASET]`
+**Usage:** `mvn verify -Pimport-dump [-Ddataset=DATASET]`
 
-#### Example:
-- `mvn verify -Pimport-dump -Ddataset=100u` - import `datasets/100u.sql.gz` dump file created using `prepare-dump.sh`.
+**For example:**
+- `mvn verify -Pgenerate-data -Ddataset=100u -Dexport-dump` will generate data based on `datasets/100u.properties` and export a database dump to a file: `datasets/100u.sql.gz`.
+- `mvn verify -Pimport-dump -Ddataset=100u` will import the database dump from a file: `datasets/100u.sql.gz`, and reboot the server(s)
 
 
 ### Run Tests
 
-Usage: `mvn verify -Ptest[,cluster] [-DrunUsers=N] [-DrampUpPeriod=SECONDS] [-DnumOfIterations=N] [-Ddataset=DATASET] [-D<dataset.property>=<value>]* [-D<test.property>=<value>]* `.
+Usage: `mvn verify -Ptest[,cluster] [-DtestParameter=value]`.
 
-_*Note:* The same dataset properties which were used for data generation/import should be supplied to the `test` phase._
+#### Common Parameters
 
-The default test `keycloak.DefaultSimulation` takes the following additional properties:
+| Parameter | Description | Default Value |
+| --- | --- | --- | 
+| `gatling.simulationClass` | Classname of the simulation to be run. | `keycloak.DefaultSimulation`  |
+| `dataset` | Name of the dataset to use. (Individual dataset properties can be overridden with `-Ddataset.property=value`.) | `default` |
+| `runUsers` | Number of users for the simulation run. | `1` |
+| `rampUpPeriod` | Period during which the users will be ramped up. (seconds) | `0` |
+| `steadyLoadPeriod` | A period of steady load. (seconds) | `30` |
+| `rampDownASAP` | When `true` the test will be checking for ramp-down condition after each *scenario step*. When `false` the check will be done only at the end of a *scenario iteration*. | `false` |
+| `pace` | A dynamic pause after each *scenario iteration*. For example if the pace is 30s and one scenario iteration takes only 20s, the simulation will wait additional 10s before continuing to the next iteration. | `0` |
```

Comment:
This *pace* parameter completely spoils the results. That means that if there were 3000 requests within 20 seconds, then 10 seconds of "pace", the result is 3000/*30* = 1000 req/s instead of 3000/*20* = 1500 req/s.